<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>copyRangeWidget</class>
 <widget class="QWidget" name="copyRangeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>189</width>
    <height>156</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>The boundaries of copying</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="labelSourceOffset">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Source offset:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="sourceOffset"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="labelTargetOffset">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Target offset:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="targetOffset"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="labelSectorsCount">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Number of sectors:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="sectorsCount"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>check_clicked(bool)</slot>
 </slots>
</ui>
